cmake/Modules/FindERTPython.cmake

```diff
@@ -23,17 +23,17 @@ if(PYTHONINTERP_FOUND)
   endif()
   set(PYTHON_INSTALL_PREFIX  "lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/${PYTHON_PACKAGE_PATH}"  CACHE STRING "Subdirectory to install Python modules in")
 
+  list(APPEND PATH_LIST "")
```

Comment:
> why do we want a blank entry in there?

I wanted to initialize `PATH_LIST`as an an empty list; that was the solution Google offered. You suggest just: 
```txt
set( PATH_LIST )
```
instead?